cobra_test.go

```diff
@@ -42,9 +42,10 @@ var cmdEcho = &Command{
 }
 
 var cmdTimes = &Command{
-	Use:   "times [# times] [string to echo]",
-	Short: "Echo anything to the screen more times",
-	Long:  `a slightly useless command for testing.`,
+	Use:        "times [# times] [string to echo]",
+	Short:      "Echo anything to the screen more times",
+	Long:       `a slightly useless command for testing.`,
+	Deprecated: true,
```

Comment:
rather than a bool, maybe this should a be a redirect string?

Deprecated: "see foobar instead"